### DISCUSSION TITLE
Error saat menggunakan function step()

### DISCUSSION THREAD

**Discussion by rulierinton** [teamalgoritma/community#119] [Q&A] [answered]
dear bapak & ibu,

saya sendang latihan menggunakan step-wise regression hanya saja saya mengalami kendala karena terjadi error seperti gambar,
apakah ada saran untuk menangani error tersebut

Thanks

![Capture](https://user-images.githubusercontent.com/101322908/170729850-d8592af6-0fff-4f69-820a-db568ac7e1a1.PNG)

**Comment by NabiilahArdini** [comment, marked as answer]
Hi @rulierinton ,

Saya menemukan diskusi error yang mirip: https://stackoverflow.com/questions/55342788/error-in-lengthobj-class-name-too-long-in-length Hasil diskusinya mengatakan bahwa mungkin error tersebut karena conflict fungsi dari package yang berbeda. Untuk itu kita perlu menyatakan bahwa fungsi step yang kita gunakan berasal dari package `stats`. 

Mungkin boleh dicoba dengan:

```
model <- stats::step(model_all, trace = 0)
```

Semoga dapat membantu Anda!
